src/test/java/io/airlift/stackfold/TestStackFolder.java

```diff
@@ -64,6 +64,112 @@ public void testFolding()
                 "\n");
     }
 
+    @Test
+    public void testThreadMxBeanDump()
+            throws Exception
+    {
+        List<StackFolding> foldings = loadFolding(readResourceLines("folding.txt"));
+
+        Set<Stack> stacks = loadStackTrace(readResourceLines("thread_mxbean_stack.txt"), foldings);
+
+        assertEquals(renderStacks(stacks), "" +
+                "testcontainers-ryuk\n" +
```

Comment:
We should probably move the expected output to a resource as well. Something like `thread_mxbean_stack.actual.txt` and `thread_mxbean_stack.expected.txt`

Reply:
Right. I picked `thread_mxbean_stack.txt` and `thread_mxbean_stack_folded.txt` names.